orchagent/portsorch.cpp

```diff
@@ -3918,3 +3918,29 @@ void PortsOrch::getPortSerdesVal(const std::string& val_str,
         lane_values.push_back(lane_val);
     }
 }
+
+void PortsOrch::increasePortNeighRefCount(const string &alias)
+{
+    Port p;
+    getPort(alias, p);
+    p.m_neigh_ref_count ++;
+    SWSS_LOG_NOTICE("Increase Port %s neigh ref count to: %d", p.m_alias.c_str(), p.m_neigh_ref_count);
```

Comment:
Should be INFO or DEBUG level

Reply:
Should be DEBUG, may create good amount of logs, if neighbour is fluctuating.